mithril-client/src/aggregator_client/http_client.rs

```diff
@@ -183,6 +211,26 @@ impl AggregatorClient for AggregatorHTTPClient {
             })
     }
 
+    async fn post_content(
+        &self,
+        url: &str,
+        json: &str,
+    ) -> Result<String, AggregatorHTTPClientError> {
+        let url = format!("{}/{}", self.aggregator_endpoint.trim_end_matches('/'), url);
+        let response = self.post(&url, json).await?;
+
+        let body =
+            response
+                .text()
+                .await
+                .map_err(|e| AggregatorHTTPClientError::SubsystemError {
+                    message: "Could not find a text body in the response.".to_string(),
+                    error: e.into(),
+                })?;
+
+        Ok(body)
```

Comment:
Maybe you could write it this way?
```suggestion
        response
            .text()
            .await
            .map_err(|e| AggregatorHTTPClientError::SubsystemError {
                message: "Could not find a text body in the response.".to_string(),
                error: e.into(),
            })
```